### PR TITLE
take es query_string instead of json

### PR DESCRIPTION
**Pull request by supernothing**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,44 +125,38 @@ File 131f95c51cc819465fa1797f6ccacf9d494aaaff46fa3eac73ae63ffbdfd8267
...
-	Observed countries: PR
-	Observed filenames: a850c62b139c87af276f4699b97ecaa9553c0d73149d635375108506fc7b34a3,128ae4a05c43d523f247340e81857eb7
-
-
-File 1c51769f1f06a512b4f96c6c12f11005a98bca31d7cb640725660110c5813d5a
-	File type: mimetype: application/x-dosexec, extended_info: PE32 executable (GUI) Intel 80386, for MS Windows
-	SHA256: 1c51769f1f06a512b4f96c6c12f11005a98bca31d7cb640725660110c5813d5a
+Found 18 matches to the search query.
+Search results for {'query': {'query_string': {'query': 'strings.domains:en.wikipedia.org AND exift
+ool.ZipFileName:AndroidManifest.xml'}}}
```
Comment (mrtizmoatwork) [inline README.md]: your query has a 3rd term that is missing from ^ this dict. Is that a problem?

File: README.md [hunk 2]
```diff
@@ -125,44 +125,38 @@ File 131f95c51cc819465fa1797f6ccacf9d494aaaff46fa3eac73ae63ffbdfd8267
 ```
 
 ```bash
-$ polyswarm_api -o /tmp/test.txt search metadata '{ "query": { "exists": {"field": "lief.libraries"}}}'
+$ polyswarm -o /tmp/test.txt search metadata "strings.domains:en.wikipedia.org AND exiftool.ZipFileName:AndroidManifest.xml AND exiftool.ZipRequiredVersion:>19"
```
Comment (mrtizmoatwork) [inline README.md]: this query differs from the json /dict blob below.

File: src/polyswarm_api/__init__.py [hunk 3]
```diff
@@ -408,13 +408,23 @@ def _fix_result(self, result):
...
         return response
 
-    async def search_query(self, query):
+    async def search_query(self, query, raw=True):
         """
         Search by Elasticsearch query using the PS API asynchronously.
 
-        :param query: Elasticsearch query
+        :param query: Elasticsearch JSON query or search string
+        :param raw: If this is a raw JSON query or needs wrapping
```
Comment (mrtizmoatwork) [inline src/polyswarm_api/__init__.py]: change to be more clear "If True, query is read in as ZYX, else it is read in as AAA"

File: src/polyswarm_api/__init__.py [hunk 4]
```diff
@@ -928,14 +938,15 @@ def search_hash(self, to_scan, hash_type='sha256'):
...
         return self.loop.run_until_complete(self.ps_api.search_hash(to_scan, hash_type))
 
-    def search_query(self, query):
+    def search_query(self, query, raw=True):
         """
         Search by Elasticsearch query using the PS API asynchronously.
 
-        :param query:
+        :param query: Elasticsearch JSON query or search string
+        :param raw: If this is a raw JSON query or needs wrapping
```
Comment (mrtizmoatwork) [inline src/polyswarm_api/__init__.py]: same as above

File: setup.py [hunk 5]
```diff
@@ -15,7 +15,7 @@ def parse_requirements():
 
 setup(
     name='polyswarm-api',
-    version='0.5.0',
+    version='0.5.1',
```
Comment (mrtizmoatwork) [inline setup.py]: you can just read this from the `_version.py` so you don't have to change it in 2 places.

File: README.md (hunk 1 above)
Comment (supernothing) [reply README.md]: added a new query, fixed

File: src/polyswarm_api/__init__.py (hunk 3 above)
Comment (supernothing) [reply src/polyswarm_api/__init__.py]: fixed

File: src/polyswarm_api/__init__.py (hunk 4 above)
Comment (supernothing) [reply src/polyswarm_api/__init__.py]: fixed

File: setup.py (hunk 5 above)
Comment (supernothing) [reply setup.py]: When I first added version, I considered doing this, but it's really hacky and would prefer to just do this.

**Pull request merged**